lib/modules/datasource/github-runners/index.ts

```diff
@@ -12,8 +12,10 @@ export class GithubRunnersDatasource extends Datasource {
     ubuntu: [{ version: '22.04' }, { version: '20.04' }, { version: '18.04' }],
     macos: [
       { version: '13' },
+      { version: '13-l' },
```

Comment:
Maintainers, can you check if this is going to work? The tests are green, but that's because my code lacks new tests, and there any no macOS tests in the `lib/modules/datasource/github-runners/index.spec.ts` file...

> `@HonkingGoose:` Are we missing MacOS and Windows tests?
>
> > `@secustor`: You are right there are currently no test for that and there should probably ones. 
> > That said the logic here is very small, the biggest junk of is here: 
> >
> > https://github.com/renovatebot/renovate/blob/7681ad00fb83b4d3404bfce24a695a2bf605693d/lib/modules/manager/github-actions/extract.spec.ts#L390

Reply:
it's safe, we use docker versioning
https://github.com/renovatebot/renovate/blob/79706e6e90e279872d2ca0299200a60674e21bd0/lib/modules/datasource/github-runners/index.ts#L40